cirq-google/cirq_google/api/v2/sweeps_test.py

```diff
@@ -147,6 +147,12 @@ def test_build_recover_const(val):
         assert val2 == val
 
 
+def test_build_covert_const_double():
+    val = 1.2355
+    val2 = v2.sweeps._recover_sweep_const(v2.run_context_pb2.ConstValue(double_value=val))
+    assert math.isclose(val, val2, abs_tol=1e-10)
```

Comment:
Should be equal if stored as double.
```suggestion
    assert val2 == val
```